src/TileSourceWebMapService.cpp

```diff
@@ -447,4 +447,13 @@ TileDataType TileSourceWebMapService::getDataType() const {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
+bool TileSourceWebMapService::isSame(TileSource const* other) const {
+  auto casted = dynamic_cast<TileSourceWebMapService const*>(other);
```

Comment:
```suggestion
  auto const* casted = dynamic_cast<TileSourceWebMapService const*>(other);
```

Reply:
Why do you prefer this one? Given the dynamic_cast, the type auto refers to is quite obvious...?

Reply:
Clang-tidy suggested it to me. I think the `*` for the pointer is already used across the project with `auto`. I didn't encounter a `const*` suggestion before though if I remember correctly.